test/routes/viewModels/taskLists/tasks/finalChecksTest.js

```diff
@@ -40,4 +43,43 @@ describe('final checks task', () => {
       expect(labels).to.contain('The offender is on remand')
     })
   })
+
+  describe('getCaProcessingAction', () => {
+    it('should show start button to serious offence question when final checks UNSTARTED', () => {
+      expect(
+        getCaProcessingAction({
+          decisions: {},
+          tasks: { finalChecks: 'UNSTARTED' },
+        })
+      ).to.eql({
+        text: 'Start now',
+        href: '/hdc/finalChecks/seriousOffence/',
+        type: 'btn',
+      })
+    })
+    it('should show change link to serious offence question when final checks DONE', () => {
+      expect(
+        getCaProcessingAction({
+          decisions: {},
+          tasks: { finalChecks: 'DONE' },
+        })
+      ).to.eql({
+        text: 'Change',
+        href: '/hdc/finalChecks/seriousOffence/',
+        type: 'link',
+      })
+    })
+    it('should show continue button to serious offence question when final checks STARTED', () => {
+      expect(
+        getCaProcessingAction({
+          decisions: {},
+          tasks: { finalChecks: 'STARTED' },
+        })
+      ).to.eql({
+        text: 'Continue',
+        href: '/hdc/finalChecks/seriousOffence/',
+        type: 'btn',
+      })
+    })
+  })
```

Comment:
Make sure the standard action shows the right values depending on task completion status